build/storage/scripts/disk_infrastructure.sh

```diff
@@ -10,195 +10,156 @@ export DEFAULT_NVME_PORT=4420
 export MAX_NUMBER_OF_NAMESPACES_ON_COTROLLER=1024
```

Comment:
IN_CONTROLLER

Reply:
Sure I will change it, this is actually how it looks right now in the main ipdk-io repo.